### ISSUE TITLE
HDF File is not loaded correct

### ISSUE THREAD

**Issue by actraiser30** [midwan/amiberry#961]
Hello

Im using amiberry 3.3 on dietpi to load a hdf file configured as uae in the hd controller not the ide one.  after updating to 4.1.6 i got this error in the photo upon loading of the hdf file

![166660287-55060c8e-f123-4db5-967d-874c6735cc6d (Medium)](https://user-images.githubusercontent.com/38106933/166897590-6ffc663a-2a63-4f5e-9394-359241dcc6f3.jpg)

if i go back and install amiberry 3.3 my hdf file works fine again

i tried various settings in amibbery 4.1.6 nothing seemed to work,any suggestion?

Thanks in advance you are doing a great work

**Comment by nemo93** [comment]
hi! I'd first recommend you upgrade to latest version either from [releases](https://github.com/midwan/amiberry/releases) or from [master](https://github.com/midwan/amiberry). You should end up with **v5.1**. 

Once you complete the upgrade launch any games/demos you have and open GUI (_F12_) > Configurations > click on **Save**. Use the `.uae` generated as a blueprint for your `.hdf`. Just make sure you change the following lines according to your setup: 

```
hardfile2=rw,DH0:/path/to/my.hdf,32,1,2,512,0,,uae0
uaehf0=hdf,rw,DH0:/path/to/my.hdf,32,1,2,512,0,,uae0
```

**Comment by midwan** [comment]
@actraiser30 
Try creating a new config from scratch with the latest version, and using the same HDF file in it (using the GUI, not manually).
Does that work?

**Comment by actraiser30** [comment]
hi there,sorry for the delay im working late,i will try with a new config and let you know if this was the issue
using the gui you mean not to autoload my hdf file and first try it manual?
thank you

**Comment by midwan** [comment]
@actraiser30 
Yes, try creating a new config from scratch using the GUI, and add your hardfile there to test.
Also, check if the same hardfile works on WinUAE - just to be sure there's nothing wrong inside that.

**Comment by symbios24** [comment]
Den katalaveno giati milame agglika,tha dokimaso efharisto

**Comment by symbios24** [comment]
Dokimasa me neo config ta esvisa ola ston fakeko,den doulepse ebgale ta idia me tin eikona.ti exi to 3.3 pou den to exoun oi neoteres ekdoseis?

**Comment by midwan** [comment]
> Den katalaveno giati milame agglika 👍 Se winuae doulevi tha dokimaso ta ipolipa efharisto

Let's keep it in English please, the project has several people who could help but they won't be able to read "greeklish"... :)

If it works on WinUAE, it _should_ work on Amiberry as well. However, I cannot recreate such an issue with any of the HDFs I have here, and I've been using them since before 3.3 even. 

Does the HDF have partitions? Custom filesystem?
How big is it? Can you zip it and send me a share link somewhere?

Feel free to join one of the Discord servers where there is an Amiberry channel, if you'd like to take this there for further details. Otherwise, if you can share the HDF you can send me a link to `midwan` at `gmail.com`.

**Comment by symbios24** [comment]
Ok no prob
I solve it,i  checked the native code option in misc and it worked,is this option enabled by default in amiberry 3.3 ??

**Comment by actraiser30** [comment]
Hello again i found a bug in Amiberry 5.2 if i enable also the fpu jit for my hdf  amiberry crashes upon loading the file
